charts/telegraf-ds/README.md

```diff
@@ -1,27 +1,23 @@
-# Telegraf-DS (DaemonSet)
+# Telegraf-DS (DaemonSet) Helm chart
 
-[Telegraf](https://github.com/influxdata/telegraf) is a plugin-driven server agent written by the folks over at [InfluxData](https://influxdata.com) for collecting & reporting metrics. This chart runs a DaemonSet of Telegraf instances to collect host level metrics for your cluster. If you need to poll individual instances of infrastructure or APIs there is a `telegraf` chart that is more suited to that usecase.
+[Telegraf](https://github.com/influxdata/telegraf) is a plugin-driven server agent used for collecting and reporting metrics. This chart runs a DaemonSet of Telegraf instances to collect host-level metrics for your cluster. To poll individual instances of infrastructure or APIs, use the [Telegraf chart](https://github.com/influxdata/helm-charts/tree/master/charts/telegraf).
 
-For this chart, Telegraf inputs cannot be customised as it aims to provide an opinionated configuration to monitor kubernetes nodes and global kubernetes monitoring.
+The Telegraf-DS Helm chart uses the [Helm](https://helm.sh) package manager to bootstrap a Telegraf-DS (DaemonSet) on a [Kubernetes](http://kubernetes.io) cluster. You cannot customize Telegraf inputs for this chart being the goal is to provide an opinionated configuration for monitoring Kubernetes nodes and global Kubernetes monitoring.
 
-## TL;DR
+## Prerequisites
+
+- Kubernetes 1.11+ with Beta APIs enabled
```

Comment:
Is PV support in underlying infrastructure an optional prerequisite for Telegraf?

Reply:
I don't see a PV option here. Can you elaborate, please?

Reply:
Noticed PV support is an optional prerequisite for Chronograf, InfluxDB, InfluxDB2, and Kapacitor, so wanted to check if same for Telegraf